packages/tokens/README.md

```diff
@@ -7,9 +7,9 @@ Direct consumption of the tokens package is optional, but recommended. The token
 
 To use the tokens, simply:
 1. Add `@department-of-veterans-affairs/mobile-tokens` to your project via your package manager (e.g. yarn)
-2. Add `import { Colors } from '@department-of-veterans-affairs/mobile-tokens` to files you wish to use them in. 
-  - Note: `Colors` are the only tokens available for now.
-3. Reference colors in your code: e.g. `buttonColor = Colors.grayMedium`
+2. Add `import { colors } from '@department-of-veterans-affairs/mobile-tokens` to files you wish to use them in. 
+  - Note: `colors` are the only tokens available for now.
+3. Reference colors in your code: e.g. `buttonColor = colors.grayMedium`
```

Comment:
```suggestion
3. Reference colors in your code: e.g. `buttonColor = colors.vadsColorGrayMedium`
```
or other more up-to-date reference.

Reply:
Still outstanding suggestion to change this to a valid color token to minimize confusion.